src/WebJobs.Script/Binding/Http/RawScriptResult.cs

```diff
@@ -90,6 +88,8 @@ public RawScriptResult(int? statusCode, object content)
                 selectedFormatter = SelectedFormatter(formatterContext);
             }
 
+            formatterContext.ContentType = response.ContentType;
```

Comment:
It was causing the SelectedFormatter to always fail, even in the catch all to string case, because the StringOutputFormatter takes Content Type into account. Since we don't care about the ContentType actually affecting which Formatter we use (that's based on data type), it makes sense to just set it after we've selected the formatter.

Reply:
It makes sense that it would take the content type into consideration (as the content type is evaluated during formatting and could potentially lead to failure), which reinforces the idea that it should be set prior to selection.

For now, it's fine to leave as is (even though it doesn't sit well), I'll track this.

Reply:
I'm going to keep it where it is for now. I can show you the behavior I was seeing in person tomorrow, if you'd like. Easy to repro.